packages/app-builder-lib/src/targets/nsis/nsisOptions.ts

```diff
@@ -231,6 +231,12 @@ export interface PortableOptions extends TargetSpecificOptions, CommonNsisOption
    * The image to show while the portable executable is extracting. This image must be a bitmap (`.bmp`) image.
    */
   readonly splashImage?: string | null
+
+  /**
+   * Disable building an universal installer of the archs specified in the target configuration
+   * @default true
+   */
+  readonly buildUniversalInstaller?: boolean
```

Comment:
Can we just move the `buildUniversalInstaller` to `CommonNsisOptions` so that it's inherited to both Nsis and Portable interfaces?

Reply:
Currently, this `buildUniversalInstaller` does not support `nsis-web`, so it is not ideal to place it globally.
![image](https://github.com/user-attachments/assets/3bbc2127-69b7-4e0d-bb88-2c8c73c25e9c)